src/System.Data.SqlClient/src/System/Data/SqlClient/SqlClientDiagnosticListenerExtensions.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Collections;
+using System.Diagnostics;
+
+namespace System.Data.SqlClient
+{
+    /// <summary>
+    /// Extension methods on the DiagnosticListener class to log SqlCommand data
+    /// </summary>
+    internal static class SqlClientDiagnosticListenerExtensions
+    {
+        public static readonly string DiagnosticListenerName = "SqlClientDiagnosticListener";
+
+        private static readonly string SqlClientPrefix = "System.Data.SqlClient.";
+        public static readonly string SqlBeforeExecuteCommand = SqlClientPrefix + nameof(WriteCommandBefore);
+        public static readonly string SqlAfterExecuteCommand = SqlClientPrefix + nameof(WriteCommandAfter);
+        public static readonly string SqlErrorExecuteCommand = SqlClientPrefix + nameof(WriteCommandError);
```

Comment:
Is there a reason these need to be `static readonly` fields, or could they be constants? e.g.:

```csharp
public const string DiagnosticListenerName = "SqlClientDiagnosticListener";

private const string SqlClientPrefix = "System.Data.SqlClient.";
public const string SqlBeforeExecuteCommand = SqlClientPrefix + nameof(WriteCommandBefore);
public const string SqlAfterExecuteCommand = SqlClientPrefix + nameof(WriteCommandAfter);
public const string SqlErrorExecuteCommand = SqlClientPrefix + nameof(WriteCommandError);
```